config_defaults_inc.php

```diff
@@ -1955,6 +1955,20 @@
  */
 $g_file_upload_method = DATABASE;
 
+/**
+ * Use Zip_attachments: zip selected file types in order to optimize performance 
+ * and save disk/db space
+ * @global integer $g_zip_attachments
+ */
+$g_zip_attachments		=	ON;
+
+/**
+ * Use Zip_excludes: define the file types to ommit from the zipping process 
+ * because the files are already compressed and/or one does not want to compress a specific file type
+ * @global integer $g_zip_excludes
```

Comment:
Same comment as above. 

Avoid long lines (try to wrap around 80, no more than 100)

`omit` takes only one M

I think some clarification on what is meant by *file types* is needed here - elsewhere in config_defauts, we speak about extensions, but here I see `x-zip_compressed` which is more a mime type, so it's a bit confusing.